Packages/manifest.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "com.unity.collab-proxy": "2.0.7",
     "com.unity.feature.2d": "2.0.0",
-    "com.unity.ide.rider": "3.0.24",
+    "com.unity.ide.rider": "3.0.25",
```

Comment:
@magictrick-dev Seems like we need to add Rider versioning to the .gitignore. I'll speak to you about this at some point. 

@Gutin1 Disregard this note lol, just putting a pin in this.